ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -425,7 +425,6 @@ open class SegmentedControl: UIView, TokenizedControlInternal {
         for (index, button) in buttons.enumerated() {
             button.updateTokenizedValues()
             button.setTitleColor(contentColor, for: .normal)
```

Comment:
If you're removing tintColor, you might also want to remove setTitleColor? They're both the old way.